.golangci.yml

```diff
@@ -45,3 +50,27 @@ linters-settings:
   revive:
     rules:
       - name: comment-spacings
+  custom:
+    kal:
+      type: "module"
+      description: KAL is the Kube-API-Linter and lints Kube like APIs based on API conventions and best practices.
+      settings:
+        linters:
+          enable:
+          - "commentstart"
+          - "conditions"
+          - "integers"
+          - "jsontags"
+          # NOTE: maxlength check currently trips over our Status fields
+          # missing a MaxLength marker
+          # - "maxlength"
+          # NOTE: we have a number of boolean fields. Should we convert them to
+          # string?
+          # - "nobools"
```

Comment:
A bunch of our OpenStack proxy bools are Just Bools(tm). We're not going to switch them.

It would be nice to enable this lint on the other parts of the API, but I'm not sure enabling this is worth all the `nolint` directives we'd need.